packit/config/common_package_config.py

```diff
@@ -172,6 +172,8 @@ class CommonPackageConfig:
         osh_diff_scan_after_copr_build: Whether to run a differential scan in
             OpenScanHub after successful Copr build.
         csmock_args: Pass additional arguments to csmock
+        use_target_repo_for_fmf_url: If the target repo should be used for fmf_url
```

Comment:
Does `fmf_url` here refer to the `test - fmf - url` field in a TF request? If so, I would rather use `fmf URL` or maybe even `test.fmf.url field in a Testing Farm request`.

Reply:
well, there's also #2044